source/wp-content/themes/wporg-developer-2023/functions.php

```diff
@@ -181,6 +185,8 @@
 // See: https://github.com/WordPress/wporg-mu-plugins/blob/trunk/mu-plugins/blocks/table-of-contents/index.php#L70
 add_filter( 'the_content', __NAMESPACE__ . '\filter_code_content', 4 );
 add_filter( 'wporg_table_of_contents_post_content', __NAMESPACE__ . '\filter_code_content' );
+add_filter( 'the_content', __NAMESPACE__ . '\filter_command_content', 4 );
+
```

Comment:
Add the filter to the table of contents hook too so that the content shows up in the table of contents.

```suggestion
add_filter( 'wporg_table_of_contents_post_content', __NAMESPACE__ . '\filter_command_content' );
```